### ISSUE TITLE
Azure Exporter support providing cloud role via Options

### ISSUE THREAD

**Issue by se02035** [census-instrumentation/opencensus-python#1052]
It is unclear how to set the cloud role value for a log record. According to https://github.com/census-instrumentation/opencensus-python/blob/5ad2a7cc62fc9663d186e15636a7b0306c9a8d7a/contrib/opencensus-ext-azure/opencensus/ext/azure/common/utils.py#L27 the cloud is always `'ai.cloud.role': os.path.basename(sys.argv[0]) or 'Python Application'` 

A correct cloud role name is essential for Azure Application Insights Application Map. In order to use another role name (which is likely true for most of the cases) a custom callback has be used to update the role name. It would be better to have an parameter on the AzureExporter(options) (https://github.com/census-instrumentation/opencensus-python/blob/1ca601584cbbd3b6019bddb9c7120c069ff4f819/contrib/opencensus-ext-azure/opencensus/ext/azure/common/__init__.py#L104) to set the cloud role (and instance) - similar to the instrumentation key/connection string,

**Comment by lzchen** [comment]
You can modify this field via [telemetry processors](https://docs.microsoft.com/en-us/azure/azure-monitor/app/api-filtering-sampling#opencensus-python-telemetry-processors)

**Comment by se02035** [comment]
Yeah, I know but wouldn't it be nicer/cleaner to have a env var here? I mean I see a couple of benefits:
1. other AI SDKs also allow to set the cloud role name via an env var.
1. Similar approach is used for passing the AI instrumentation key
1. the telemetry processor has to be configured for every AzureExporter/AzureLogHandler instance. E..g you need to do something like this (when using Azure Functions): 
    - `OpenCensusExtension._exporter.add_telemetry_processor(..)` plus use the same for other AzureLogHandler (if available).

**Comment by lzchen** [comment]
@se02035 

Those are excellent points and I agree with it. Feel free to open a PR and I will review it :)

**Comment by se02035** [comment]
@lzchen , created a PR for allowing devs to set the cloud role name via an envvar. https://github.com/census-instrumentation/opencensus-python/pull/1061
